src/components/cards/DataSummaryCardGroup.less

```diff
@@ -0,0 +1,51 @@
+@import '../../css/base.less';
+
+.data-summary-card-group {
+  display: flex;
+}
+  
+.data-summary-card-group__connected-card,
+.data-summary-card-group__separated-card {
+  background-color: #ffffff;
+  flex-grow: 1;
+  flex-basis: 0;
+  position: relative;
+}
+
+.data-summary-card-group__separated-card {
+  margin-left: 15px;
+  &:first-child {
+    margin-left: 0px;
+  }
+}
+
+.data-summary-card-group__connected-card {
+  margin: 0;
+  &:first-child {
+    border: none;
+  }
+
+  .left-border {
```

Comment:
With BEM we are trying to avoid nesting classes - I think you can move everything in `.left-border` (here and on line 45) to the top level of the class anyway, and then just have `&:first-child` take care of eliminating the border for the first children